src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1469,7 +1711,7 @@ public C(string s)
         public async Task TestLambdaWithIncorrectNumberOfParameters()
         {
             await TestMissingInRegularAndScriptAsync(
-@"
+        @"
```

Comment:
this file seems to have been formatted with a strange set of rules.  can you please undo those changes?  thanks!